fl4health/utils/dataset.py

```diff
@@ -53,3 +53,31 @@ def __init__(
         self.targets = mnist_dataset.targets
         self.transform = transform
         self.target_transform = target_transform
+
+
+class Cifar10Dataset(BaseDataset):
+    def __init__(
+        self,
+        data_path: Path,
+        train: bool,
+        transform: Union[None, Callable] = None,
+        target_transform: Union[None, Callable] = None,
+    ):
+        cifar10_dataset = CIFAR10(data_path, train=train, download=True)
+        self.data = torch.from_numpy(cifar10_dataset.data)
+        self.targets = torch.Tensor(cifar10_dataset.targets).long()
```

Comment:
Interesting that this long conversion is necessary for Cifar but not Mnist 🤔

Reply:
Actually the long conversion is not necessary, but unlike Mnist, we do need to convert the targets to tensors for Cifar.

Reply:
My bad... Turns out we do need to convert it to long